static/script/devices/mediaplayer/samsung_streaming.js

```diff
@@ -663,10 +703,28 @@ require.def(
                 var mime = this._mimeType.toLowerCase();
                 return mime === "application/vnd.apple.mpegurl" || mime === "application/x-mpegurl";
             },
+            
+            _isCurrentTimeInRangeTolerance: function (seconds) {
```

Comment:
Only used by code I suggested be removed and replaced with the 8 seconds interval updating seekable range.